integration-cli/docker_cli_run_test.go

```diff
@@ -2804,10 +2804,12 @@ func (s *DockerSuite) TestRunContainerWithReadonlyRootfsWithAddHostFlag(c *check
 
 func (s *DockerSuite) TestRunVolumesFromRestartAfterRemoved(c *check.C) {
 	prefix := ""
+	slash := `/`
```

Comment:
I don't think that works, because (iiuc) in this test it should check for the platform the *daemon* runs on, which may be a different platform than where the client/tests run